src/builder.rs

```diff
@@ -32,20 +41,22 @@
 //! ```
 
 use hal;
+use hal::digital::OutputPin;
 
 use crate::displayrotation::DisplayRotation;
 use crate::displaysize::DisplaySize;
-use crate::interface::I2cInterface;
+use crate::interface::{I2cInterface, SpiInterface};
 use crate::mode::displaymode::DisplayMode;
 use crate::mode::raw::RawMode;
 use crate::properties::DisplayProperties;
 
 /// Builder struct. Driver options and interface are set using its methods.
 #[derive(Clone, Copy)]
-pub struct Builder {
+pub struct Builder<CS=NoOutputPin> {
     display_size: DisplaySize,
     rotation: DisplayRotation,
     i2c_addr: u8,
+    spi_cs: CS,
```

Comment:
Why not make this an option instead?

Reply:
By using a type for the "optionalness" it should be more or less guaranteed that there's no runtime overhead at runtime (the CS pin manipulation expands to nothing). With an Option I'm not sure if that's the case.

Reply:
Ah I see, yeah that makes sense with a defaulted type param. Let's leave it be then :)